File: app/src/main/java/com/riders/thelab/data/remote/dto/ArtistsResponseJsonAdapter.java
package com.riders.thelab.data.remote.dto;

import com.squareup.moshi.FromJson;
import com.squareup.moshi.ToJson;

import java.util.List;

public class ArtistsResponseJsonAdapter {

    @FromJson
    List<Artist> artistsFromJson(List<Artist> artistsEventJson) {
        return artistsEventJson;
    }

    @ToJson
    String artistsToJson(List<Artist> artists) {
        return artists.toString();
    }
}

File: core/data/src/main/java/com/riders/thelab/core/data/remote/dto/artist/ArtistsResponseJsonAdapter.kt
package com.riders.thelab.core.data.remote.dto.artist

import com.squareup.moshi.FromJson
import com.squareup.moshi.ToJson

class ArtistsResponseJsonAdapter {
    @FromJson
    fun artistsFromJson(artistsEventJson: List<Artist>): List<Artist> {
        return artistsEventJson
    }

    @ToJson
    fun artistsToJson(artists: List<Artist?>): String {
        return artists.toString()
    }
}